### ISSUE TITLE
Comma rule is triggered when there are comments following the comma

### ISSUE THREAD

**Issue by mariusc** [realm/SwiftLint#2362]
Hi,

I found some cases where the comma rule is triggered, but I don't think it should :). The common usecase for this is inside an array or dictionary, where the values are declared one per line, and you add a comment after one of them.

![image](https://user-images.githubusercontent.com/844175/44568887-3533b100-a781-11e8-9e9b-070bb21a89d0.png)

### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues) 
I found some for comma rule in image literals and in strings, but that is not the case here

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
mariusconstantinescu:~$ swiftlint version
0.27.0
mariusconstantinescu:~$ echo ""User-Agent": shouldSendCustomUserAgent ? "customUserAgent" : "standardUserAgent",// check which user agent value to send" | swiftlint lint --use-stdin
<nopath>:1:76: warning: Comma Spacing Violation: There should be no space before and one after any comma. (comma)
Done linting! Found 1 violation, 0 serious in 1 file.
mariusconstantinescu:~$ 
```

### Environment

* SwiftLint version: 
*0.27.0*
* Installation method used: 
*Homebrew*
* Paste your configuration file: 
*no custom configuratin file*
* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
*No*
* Which Xcode version are you using (check `xcode-select -p`)?
*Version 9.4 (9F1027a)*
* Do you have a sample that shows the issue? 

```swift
let customHeaders1 = [
		"User-Agent": shouldSendCustomUserAgent ? "customUserAgent" : "standardUserAgent",		// check which user agent value to send
		"Authorization": "userToken"
	]
```

![image](https://user-images.githubusercontent.com/844175/44568400-45e32780-a77f-11e8-8fd5-c1a50a38b2df.png)
